### ISSUE TITLE
Section on managing focus using `aria-activedescendant` should be updated

### ISSUE THREAD

**Issue by mayank99** [w3c/aria-practices#2048]
The article on [managing focus using `aria-activedescendant`](https://w3c.github.io/aria-practices/#kbd_focus_activedescendant) only mentions the `textbox` role when the value is not an owned element.

https://github.com/w3c/aria-practices/blob/44a66c9e7205d980887e1d4ede55e41af6873d24/aria-practices.html#L5647

The [spec for `aria-activedescendant`](https://w3c.github.io/aria/#aria-activedescendant) (which I'm assuming is the correct source) also mentions `combobox` and `searchbox`.

![image](https://user-images.githubusercontent.com/9084735/135172437-0077eab2-ca90-447a-96d0-933c572c4316.png)

The article should be updated to mention those roles, or even streamlined to not mention that info explicitly but rather point readers to the spec.

**Comment by mcking65** [comment]
@mayank99, thank you for the feedback. This issue will be resolved in the new version of APG that will be published on May 19, 2022. Note, it will be in a new location to be announced on May 19.
